.github/workflows/development.yaml

```diff
@@ -8,6 +8,9 @@ jobs:
     strategy:
       matrix:
         include:
+          - distro: alpine
+            py_ver: '3.10'
+            platform: linux/amd64
```

Comment:
Can you update the `if` conditions here to align with the ones we've been applying upstream in the image stack? Similar to:

```yaml
    if: |
      github.event_name == 'push' &&
      github.ref == 'refs/heads/master' &&
      (
        github.repository_owner == 'datajoint' ||
        github.repository_owner == 'datajoint-company'
      )
```